app/portainer/components/Base/Buttons/Button/Button.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import { Meta, Story } from '@storybook/react';
+import { PropsWithChildren } from 'react';
+
+import { Button, Props } from './Button';
+
+export default {
+  component: Button,
+  title: 'Components/Buttons/Button',
+} as Meta;
+
+function Template({
+  onClick,
+  type,
```

Comment:
I think we can remove type from props.

Reply:
also children IMO

Reply:
agree with removing type, but removing children results in an empty button for the default story

Reply:
of course. replace children (in the story) with either:
icon and label, or just label.

story arguments don't need to map 100% to the component props

Reply:
but doing that, for example in the `Template` function, results in every single button story showing the same label, adding it as a prop allows for different labels, I get that it's not that important but it's better for the documentation I believe

Reply:
wait I think my last comment isn't right, checking...

Reply:
I pushed the change now, I can see it's much cleaner without the children prop, I removed it from the ButtonGroup as well